compose/ui/ui-uikit/src/nativeInterop/cinterop/utils.def

```diff
@@ -1,3 +1,5 @@
 linkerOpts = -framework UIKit
 package = androidx.compose.ui.uikit.utils
-language = Objective-C
\ No newline at end of file
+language = Objective-C
+compilerOpts = -D_Float16=short
```

Comment:
This is indeed a workaround for https://youtrack.jetbrains.com/issue/KT-69094, but to build with Xcode 16 and Kotlin < 2.0.20 this will be required. It should do no harm and we are shipping an equivalent workaround in 2.0.20 in the default K/N clang args

Reply:
Ok! Sounds reasonable.